CMakeLists.txt

```diff
@@ -281,6 +281,7 @@ elseif(CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
   target_compile_options(_WarningsAsError INTERFACE
     /WX
 	/wd4996
+  /we4596
```

Comment:
I was going to write indentation, but I actually am coming around to indenting every line with different indentation

Reply:
🙄 🤦